.github/workflows/build-resctl-demo.yml

```
@@ -6,14 +6,19 @@ on:
   # see https://github.com/iocost-benchmark/iocost-benchmarks-ci/issues/1
   #
   # every Monday at 01:00
+  push:
   schedule:
   - cron: "0 1 * * 1"
   workflow_dispatch:
 
 jobs:
-  run:
+  build:
     name: Build resctl-demo
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version:
+          [{ minor: "v2.1", ref: "v2.1.2" }, { minor: "v2.2", ref: "v2.2.0" }]
```

```suggestion
          - { minor: "v2.1", ref: "v2.1.2" }
          - { minor: "v2.2", ref: "v2.2.0" }
```
would syntax like this make it more readable ?

---

Much more readable indeed!